src/util/accessibility/live.ts

```diff
@@ -39,7 +39,7 @@ export class Live implements OnDestroy {
 
   ngOnDestroy() {
     const element = getLiveElement(this._document);
-    if (element) {
+    if (element != null && element.parentElement != null) {
```

Comment:
Is there a case where element is not null and element.parentElement is null ? If not, I would probably add a `element.parentElement !.removeChild(...`  on the next line.

Reply:
we're always attaching it to body, so parent element is always body if I'm not mistaken

Reply:
Except if a developer already created a `<div id="ngb-live">` in his app, no ?

Reply:
> Is there a case where element is not null and element.parentElement is null

I don't know if it's the case here, but typically _detached node_ would be an example of that.

Reply:
Yes, sure, I said it because of the specific getLiveElement implementation : querySelector is done on the body, and attached to the body if it doesn't exist.